docs/manual/exporting-the-schedule/index.md

```diff
@@ -252,15 +252,15 @@ If you have secure shell access (SSH) to the remote web server, you could write
 Website widgets
 ---------------
 
-Example HTML, Javascript and CSS code for your public website are provided in the *widgets* folder of the Airtime installation tarball, or on GitHub: <https://github.com/sourcefabric/Airtime/tree/master/widgets>
+Example HTML, Javascript and CSS code for your public website are provided in the *widgets* folder of the Libretime installation tarball, or on GitHub: <https://github.com/sourcefabric/Libretime/tree/master/widgets>
```

Comment:
This link doesn't work, I can't find where any example widgets are any more

Reply:
Here they are - https://github.com/LibreTime/libretime/tree/master/airtime_mvc/public/widgets

Reply:
Will be in next PR.

Reply:
You should be able to just make the edit and push the commit on this PR, let me know if you need any help with the appropriate git command.

Reply:
I think I edited and pushed it right? Let me know if not. I'm a noob.